Example/Tests/Mocks/MockedRequest.swift

```diff
@@ -14,16 +14,19 @@ class MockedRequest: Request {
     var query: RequestQuery?
     var headers: RequestHeaders?
     var body: Any?
+    var cachePolicy: URLRequest.CachePolicy
     
     init(url: URL?,
          method: RequestMethod = .get,
          query: RequestQuery? = nil,
          headers: RequestHeaders? = nil,
-         body: Any? = nil) {
+         body: Any? = nil,
+         cachePolicy: URLRequest.CachePolicy = .useProtocolCachePolicy) {
```

Comment:
Moet dat verplicht meegegeven worden? kan het zijn dat ik gewoon geen cachPolicy wil instellen en dus ook niet de `useProtocolCachePolicy`

Reply:
Ik vind dat het aan de app is om dat te definieren. Ik doe liever geen default implementaties in Cara. Je kan dit dus altijd doen in de app:

```swift
extension Request {
    var cachePolicy: URLRequest.CachePolicy {
        return .reloadIgnoringLocalCacheData
    }
}
```